### PR TITLE
New Home Page

### PR DESCRIPTION
**Pull request by robbiejaeger**
Experimenting with a new look for the FE-curriculum home page.

Here is a WIP of a splash image for the front page. I'm still torn on what to do for the remaining links needed on that page:
* Calendar (still need a link to this since it's in the top nav?)
* Lessons (still need a link to this since it's in the top nav?)
* Projects (still need a link to this since it's in the top nav?)

* Professional Development
* Mentor and Posse Info
* Pairing Sign-Up Sheet
* Job Search Resources
* Mod 0 Capstone 

Would love some direction or to see some spikes on this. How do we put the links on the page in a nice way?

This branch also contains some general cleanup of unused items in the curriculum site.

---

*Desktop View:*
Desktop

--- 

*Mobile View:*
Mobile

### CODE REVIEW

**Comment by letakeane** [comment]
Maybe for the remaining links, the three sections (Daily Turing, Your Resources, Before & After Turing) could be turned into columns, with the buttons arrayed vertically?

Although Before & After Turing links aren't especially high priority and could be "below the fold" ....

I think the Calendar, Lessons, and Projects links should still be there, even though they're also in the top nav.

**Comment by brittanystoroz** [comment]
Ooooh looks fancy

File: _sass/home-page.scss [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  p {
+    font-family: $standard-copy, sans-serif;
+    font-size: 1.1em;
+    margin: 0px 0px 0px 80px;
+  }
+}
+
+@media (max-width: $tablet-width) {
+  .splash {
+    height: 350px;
```
Comment (brittanystoroz) [inline _sass/home-page.scss]: I think this might look better at 400px, what do you think?

**Comment by robbiejaeger** [comment]
I’ve added some additional sections to the page to include some links to resources. Links I have removed to simplify and reduce unnecessary clutter on the page:

* Mentor and Posse Info - no longer used because of mentor assignments
* Pairing Sign-Up Sheet - no longer used because of Paired app
* Job Search Resources - link outdated and not used
* Mod 0 Capstone - this is now it’s own website maintained by someone else, not completely sure yet

I also added a short section featuring the Paired app - seems to be well maintained and used by students.

image
image

**Comment by ameseee** [comment]
I think this looks great @robbiejaeger! 

What do you think about using the same `lighten($turing-secondary, 25%);` on the hover/active states for the nav items, same color as the .`lesson-segment` boxes?

image

**Comment by robbiejaeger** [comment]
Added the mentor list back in because Kayt said it’s still used. Also updated the version of jQuery we use vie the CDN link.

image
image

**Comment by robbiejaeger** [comment]
Louisa gave the feedback that the “Paired” section is really big. Do we want to feature it this much? We can probable make this more subtle in future iterations, but I reduced the image size in the meantime.

**Pull request merged**